provisionersdk/proto/provisioner.proto

```diff
@@ -175,6 +192,7 @@ message Parse {
         string directory = 1;
     }
     message Complete {
+        repeated TemplateVariable template_variables = 1;
         repeated ParameterSchema parameter_schemas = 2;
```

Comment:
@kylecarbs is this safe or should we leave `1` empty and add `template_variables` as `3`?

Reply:
I think it should be fine.

Customers aren't running provisioner daemons externally at the moment, so this incompatibility should be alright.